applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -352,6 +359,10 @@ config LE_AUDIO_MSG_SUB_STACK_SIZE
 	int "Stack size for LE Audio subscriber"
 	default 2048
 
+config BT_MGMT_MSG_SUB_STACK_SIZE
+	int "Stack size for bt_mgmt subscriber"
+	default 2048
```

Comment:
Did you check if this is roughly the right amount?

Reply:
I use the same setting like above, let me use thread anaylzer for the check.